app/controllers/posts_controller.rb

```diff
@@ -1,6 +1,6 @@
 class PostsController < ApplicationController
   def index
-    @user = User.find(params[:user_id])
+    @user = User.includes(posts: { comments: :user }).find(params[:user_id])
```

Comment:
- [ ] according to the requirements you should add screenshots of the logs before and after the fix to the PR description.

Reply:
This issue is well resolved! ✔️